python/pyspark/errors/exceptions/connect.py

```diff
@@ -16,7 +16,7 @@
 #
 import pyspark.sql.connect.proto as pb2
 import json
-from typing import Dict, List, Optional, TYPE_CHECKING
+from typing import Dict, List, Optional, TYPE_CHECKING, overload
```

Comment:
```suggestion
from typing import Dict, List, Optional, TYPE_CHECKING
```

Linter complains :-).

Reply:
Done